### DISCUSSION TITLE
How to export Dataset to grib2 file?

### DISCUSSION THREAD

**Discussion by zxdawn** [pydata/xarray#7367] [Q&A] [answered]
I found this question was asked both on [StackOverflow](https://stackoverflow.com/q/68486876/7347925) and [cfgrib](https://github.com/ecmwf/cfgrib/issues/18) repo.

It seems `to_grib` isn't available in xarray anymore. Is there any new method to convert the data back to grib2?

**Comment by zxdawn** [comment, marked as answer]
well, the function name is different. Here's the solution:
```
from cfgrib.xarray_to_grib import to_grib
to_grib(ds, 'out.grib')
```

**Reply by n-beckage** [reply]
Is there a way to save multiple xarray datasets to a single grib file? Looking for something with the reverse functionality of `cfgrib.open_datasets(file)` , which returns a list of all the xarray datasets contained in file. In contrast, I'd like to write multiple xarray datasets to a single grib file.

**Reply by TomNicholas** [reply]
Do you mean writing each dataset to a different group? Or somehow combining the datasets before writing the result?
